### ISSUE TITLE
Cannot import lmdb "/FIPython.h: No such file or directory"

### ISSUE THREAD

**Issue by siebz0r** [jnwatson/py-lmdb#136]
### Affected Operating Systems
- Linux
### Affected py-lmdb Version

0.92
### py-lmdb Installation Method

sudo pip install lmdb==0.92
### Using bundled or distribution-provided LMDB library?

Bundled
### Distribution name and LMDB library version

Ubuntu 16.04
### Machine "free -m" output

```
              total        used        free      shared  buff/cache   available
Mem:          15933        4110        2458         942        9364       10684
Swap:             0           0           0
```
### Other important machine info

Running in docker.
### Describe Your Problem

As stated in the title, I cannot import lmdb. (I thus also cannot perform some of the commands listed in this template.)
### Errors/exceptions Encountered

```
x86_64-linux-gnu-gcc: error: /FIPython.h: No such file or directory
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/lmdb/__init__.py", line 42, in <module>
    from lmdb.cpython import *
ImportError: No module named 'lmdb.cpython'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/distutils/unixccompiler.py", line 118, in _compile
    extra_postargs)
  File "/usr/lib/python3.5/distutils/ccompiler.py", line 909, in spawn
    spawn(cmd, dry_run=self.dry_run)
  File "/usr/lib/python3.5/distutils/spawn.py", line 36, in spawn
    _spawn_posix(cmd, search_path, dry_run=dry_run)
  File "/usr/lib/python3.5/distutils/spawn.py", line 159, in _spawn_posix
    % (cmd, exit_status))
distutils.errors.DistutilsExecError: command 'x86_64-linux-gnu-gcc' failed with exit status 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/cffi/ffiplatform.py", line 58, in _build
    dist.run_command('build_ext')
  File "/usr/lib/python3.5/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.5/distutils/command/build_ext.py", line 338, in run
    self.build_extensions()
  File "/usr/lib/python3.5/distutils/command/build_ext.py", line 447, in build_extensions
    self._build_extensions_serial()
  File "/usr/lib/python3.5/distutils/command/build_ext.py", line 472, in _build_extensions_serial
    self.build_extension(ext)
  File "/usr/lib/python3.5/distutils/command/build_ext.py", line 532, in build_extension
    depends=ext.depends)
  File "/usr/lib/python3.5/distutils/ccompiler.py", line 574, in compile
    self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
  File "/usr/lib/python3.5/distutils/unixccompiler.py", line 120, in _compile
    raise CompileError(msg)
distutils.errors.CompileError: command 'x86_64-linux-gnu-gcc' failed with exit status 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.5/dist-packages/lmdb/__init__.py", line 48, in <module>
    from lmdb.cffi import *
  File "/usr/local/lib/python3.5/dist-packages/lmdb/cffi.py", line 364, in <module>
    library_dirs=_config_vars['extra_library_dirs'])
  File "/usr/local/lib/python3.5/dist-packages/cffi/api.py", line 437, in verify
    lib = self.verifier.load_library()
  File "/usr/local/lib/python3.5/dist-packages/cffi/verifier.py", line 113, in load_library
    self._compile_module()
  File "/usr/local/lib/python3.5/dist-packages/cffi/verifier.py", line 210, in _compile_module
    outputfilename = ffiplatform.compile(tmpdir, self.get_extension())
  File "/usr/local/lib/python3.5/dist-packages/cffi/ffiplatform.py", line 29, in compile
    outputfilename = _build(tmpdir, ext, compiler_verbose, debug)
  File "/usr/local/lib/python3.5/dist-packages/cffi/ffiplatform.py", line 65, in _build
    raise VerificationError('%s: %s' % (e.__class__.__name__, e))
cffi.ffiplatform.VerificationError: CompileError: command 'x86_64-linux-gnu-gcc' failed with exit status 1
```
### Describe What You Expected To Happen

I expected that lmdb could be imported.
### Describe What Happened Instead

The exception stated above.
### To reproduce

Build and run the following Dockerfile:

```
FROM ubuntu:16.04

RUN apt-get update && apt-get install -y \
    build-essential \
    python3.5 \
    python3.5-dev \
    wget

RUN wget -qO- https://bootstrap.pypa.io/get-pip.py | python3.5 && \
    pip install pip==8.1.2 --upgrade

RUN pip install lmdb==0.92

ENTRYPOINT ["python3.5", "-c", "import lmdb"]
```

**Comment by dw** [comment]
Hi,

Thanks for catching this. "Somehow" a .egg got uploaded for Windows that was marked architecture-independent, and included a Windows-specific _config.py.

I had to rejig the Windows binary builds in this release because PyPI stopped accepting uploads over non-SSL connections, so clearly I've broken something.

I deleted the weird looking .egg and things seem to work again. Thanks for the report, going to keep this bug open so I don't forget about it during next release.
